assets/app/views/create/nextSteps.html

```diff
@@ -46,9 +46,9 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
         <p>You may want to use the <code>oc</code> command line tool to help with troubleshooting. After <a target="_blank" href="{{'get_started_cli' | helpLink}}">downloading and installing</a> it, you can log in, switch to this particular project, and try some commands :</p>
 
         <pre class="code prettyprint">oc login {{loginBaseUrl}}
-  oc project {{projectName}}
-  oc build-logs -h
-  oc logs -h</pre>
+oc project {{projectName}}
+oc build-logs -h
+oc logs -h</pre>
```

Comment:
This is fine, but wondering if we should make this a `div` with `white-space: pre-line;` instead. Then we don't have to be so careful with leading space.